suite-native/module-accounts-import/src/components/AccountImportDetail.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { useNavigation } from '@react-navigation/core';
+
+import { setOnboardingFinished } from '@suite-native/module-settings';
+import {
+    AccountsImportStackParamList,
+    AccountsImportStackRoutes,
+    AppTabsRoutes,
+    HomeStackRoutes,
+    RootStackParamList,
+    RootStackRoutes,
+    StackToTabCompositeScreenProps,
+} from '@suite-native/navigation';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Box, Button, Divider, Text } from '@suite-native/atoms';
+import { Form, useForm } from '@suite-native/forms';
+import { AccountsRootState, selectAccountsByNetworkAndDevice } from '@suite-common/wallet-core';
+import { yup } from '@trezor/validation';
+import { networks, NetworkSymbol } from '@suite-common/wallet-config';
+import { AccountInfo } from '@trezor/connect';
+
+import { HIDDEN_DEVICE_ID, HIDDEN_DEVICE_STATE, importAccountThunk } from '../accountsImportThunks';
+import { AccountImportOverview } from './AccountImportOverview';
+
+type AccountImportDetailProps = {
+    networkSymbol: NetworkSymbol;
+    accountInfo: AccountInfo;
+};
+
+const importAnotherButtonStyle = prepareNativeStyle(utils => ({
+    marginTop: utils.spacings.large,
+    marginBottom: utils.spacings.medium,
+}));
+const contentWrapperStyle = prepareNativeStyle(_ => ({
+    width: '100%',
+    flex: 1,
+    justifyContent: 'flex-end',
+}));
+
+const accountImportFormValidationSchema = yup.object({
+    accountLabel: yup.string().required().max(30),
+});
+type AccountImportFormValues = yup.InferType<typeof accountImportFormValidationSchema>;
+
+type NavigationProp = StackToTabCompositeScreenProps<
+    AccountsImportStackParamList,
+    AccountsImportStackRoutes.AccountImport,
+    RootStackParamList
+>;
+
+export const AccountImportDetail = ({ networkSymbol, accountInfo }: AccountImportDetailProps) => {
+    const { applyStyle } = useNativeStyles();
+    const dispatch = useDispatch();
+    const navigation = useNavigation<NavigationProp>();
+    const deviceNetworkAccounts = useSelector((state: AccountsRootState) =>
+        selectAccountsByNetworkAndDevice(state, HIDDEN_DEVICE_STATE, networkSymbol),
+    );
+    const defaultAccountLabel = `${networks[networkSymbol].name} #${
+        deviceNetworkAccounts.length + 1
+    }`;
+    const form = useForm<AccountImportFormValues>({
+        validation: accountImportFormValidationSchema,
+        defaultValues: {
+            accountLabel: defaultAccountLabel,
+        },
+    });
+    const {
+        handleSubmit,
+        formState: { errors },
+    } = form;
+
+    const handleImportAccount = ({ accountLabel }: AccountImportFormValues) => {
+        if (accountInfo) {
```

Comment:
nit

```tsx
if (!accountInfo) return;
```

Reply:
Actually, there is no need to have this check since I moved it to this component because it's already conditionally rendered based on `accountInfo`, so I removed this check [17d9640](https://github.com/trezor/trezor-suite/pull/6591/commits/17d96404cc430378c2b939def692908d282ab316)